full_fov_phased_rx.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/python
+
+"""
+    full_fov
+    ~~~~~~~~
+    The mode transmits with a pre-calculated phase progression across the array which illuminates
+    the full FOV, and receives on all antennas. The first pulse in each sequence starts on the 0.1
+    second boundaries, to enable bistatic listening on other radars.
+
+    :copyright: 2022 SuperDARN Canada
+    :author: Remington Rohel
+"""
+
+import numpy as np
+from experiment_prototype.experiment_utils.sample_building import get_phase_shift
+
+import borealis_experiments.superdarn_common_fields as scf
+from experiment_prototype.experiment_prototype import ExperimentPrototype
+
+
+def rx_phase_pattern(beam_angle, freq_khz, antenna_count, antenna_spacing, offset=0.0):
+
+    xcf_directions = {
+        10400: [-28.8, -23.96, -19.92, -14.68, -11.24, -7.4, -3.48, -1.36,
+                1.36, 3.48, 7.5, 11.24, 14.68, 19.92, 23.96, 28.8],
+        10500: [-28.8, -23.86, -20.02, -14.78, -11.24, -7.5, -3.53, -1.41,
+                1.41, 3.53, 7.5, 11.24, 14.78, 20.02, 23.86, 29.],
+        10600: [-29., -23.76, -20.02, -14.78, -11.24, -7.5, -3.48, -1.32,
+                1.32, 3.48, 7.6, 11.24, 14.78, 20.02, 23.76, 29.],
+        10700: [-29.1, -23.76, -20.12, -14.88, -11.24, -7.6, -3.53, -1.32,
+                1.32, 3.53, 7.6, 11.24, 14.88, 20.12, 23.76, 29.2],
+        10800: [-29.3, -23.76, -20.12, -14.98, -11.24, -7.6, -3.53, -1.32,
+                1.32, 3.53, 7.7, 11.24, 14.98, 20.12, 23.76, 29.4],
+        10900: [-29.3, -23.66, -20.12, -15.08, -11.24, -7.7, -3.56, -1.32,
+                1.32, 3.56, 7.7, 11.24, 15.08, 20.12, 23.66, 29.4],
+        12200: [-28.4, -22.26, -17.62, -14.78, -11.24, -8.15, -4.96, -1.82,
+                1.82, 4.96, 8.15, 11.24, 14.78, 17.62, 22.26, 28.5],
+        12300: [-28.5, -22.46, -17.62, -14.78, -11.24, -8.1, -4.96, -1.82,
+                1.82, 4.96, 8.1, 11.24, 14.78, 17.62, 22.46, 28.6],
+        12500: [-28.7, -22.56, -17.62, -14.69, -11.24, -7.9, -4.88, -1.92,
+                1.92, 4.88, 7.9, 11.24, 14.69, 17.62, 22.56, 28.8],
+        13000: [-29.3, -22.86, -17.72, -14.58, -11.14, -7.6, -4.96, -2.12,
+                2.12, 4.96, 7.6, 11.14, 14.58, 17.72, 22.86, 29.4],
+        13100: [-29.6, -22.96, -17.82, -14.48, -11.34, -7.55, -4.93, -2.12,
+                2.12, 4.93, 7.55, 11.34, 14.48, 17.82, 22.96, 29.7],
+        13200: [-29.6, -23.06, -17.92, -14.48, -11.24, -7.6, -4.96, -2.12,
+                2.12, 4.96, 7.6, 11.24, 14.48, 17.92, 23.06, 29.7],
+        }
+
+    return get_phase_shift(xcf_directions[int(freq_khz)], freq_khz, antenna_count,
+                           antenna_spacing, offset)
+
+
+class FullFOV(ExperimentPrototype):
+    def __init__(self, **kwargs):
+        """
+        kwargs:
+
+        freq: int
+
+        """
+        cpid = 3800
```

Comment:
This CPID collides with `full_fov.py`. I think the best course of action would be to save this file as `full_fov.py`

Reply:
I have moved full_fov_phased_rx to become the new full_fov experiment